opendm/context.py

```diff
@@ -23,15 +23,15 @@
 # define orb_slam2 path
 orb_slam2_path = os.path.join(superbuild_path, "src/orb_slam2")
 
-# define smvs join_paths
+# define mve join_paths
 makescene_path = os.path.join(superbuild_path, 'src', 'elibs', 'mve', 'apps', 'makescene', 'makescene') #TODO: don't install in source
-smvs_path = os.path.join(superbuild_path, 'src', 'elibs', 'smvs', 'app', 'smvsrecon')
+mve_path = os.path.join(superbuild_path, 'src', 'elibs', 'mve', 'apps', 'dmrecon', 'dmrecon')
+mve_path_pc = os.path.join(superbuild_path, 'src', 'elibs', 'mve', 'apps', 'scene2pset', 'scene2pset -F2')
```

Comment:
`-F2` should not be hard coded as it references the chosen scale value (depends on `--max-pixels`). If a dataset has really large input images (or really small images), this will fail. I think it can be looked up by looking at the contents of the `.mve` folders.

Reply:
Another alternative is to calculate a scale value ourselves instead of letting MVE sort it out and pass it to both scene2pset (via -F) and to dmrecon (via -s).

Reply:
Maybe this is why you noticed a few failures during your tests.